lib/importers/tpas_satisfaction/importer.rb

```diff
@@ -0,0 +1,23 @@
+module Importers
+  module TpasSatisfaction
+    class Importer
+      def initialize(
+        retriever: Retriever,
+        record: Record,
+        saver: Saver,
+        config: Rails.configuration.x.tpas
+      )
+        @retriever = retriever.new(config: config)
+        @record = record
+        @saver = saver
+      end
+
+      def import
+        @retriever.process_emails do |email|
+          records = @record.build(io: email.file)
+          records && @saver.new(records: records).save
```

Comment:
Feels like perhaps the `saver` should just noop if records are `nil` i.e. we push that logic down, so we could drop the boolean expression here.